source/frontend/src/components/maps/leaflet/LayerPopup/LayerPopupView.test.tsx

```diff
@@ -227,6 +227,111 @@ describe('LayerPopupView component', () => {
       expect(history.location.pathname).toBe('/mapview/sidebar/research/new');
     });
 
+    it('only shows research file option if file disposed or retired', async () => {
+      const { getByTestId, getByText, queryByText } = setup({
+        layerPopup: {
+          latlng: undefined,
+          layers: [],
+        },
+        featureDataset: null,
+
+        claims: [Claims.RESEARCH_ADD],
+      });
+      const ellipsis = getByTestId('fly-out-ellipsis');
+      await act(async () => userEvent.click(ellipsis));
+      const link = getByText('Research File');
+      expect(queryByText('Acquisition File')).toBeNull();
+      expect(queryByText('Disposition File')).toBeNull();
+      expect(queryByText('Lease/Licence File')).toBeNull();
+      expect(link).toBeVisible();
+    });
+
+    it('only shows research file option if file disposed or retired', async () => {
```

Comment:
These two tests have the same title/description so not sure which one is which - please update to reflect actual test case